stories/webapp/hierarchy/HierarchyFolderRow.js

```diff
@@ -10,18 +10,16 @@ export const HierarchyFolderRow = ({ childCount, children, open }) => (
     name={faker.commerce.department()}
     open={open}
     showToggle
+    style={{ minWidth: '320px' }}
```

Comment:
This min-width allows us to handle hierarchy levels that go way beyond the screen width. It doesn't show a whole role of data but at least it doesn't break.

You can test this in the storybook.